src/pke/unittest/utbgvrns/UnitTestBGVrns.cpp

```diff
@@ -141,84 +141,103 @@ constexpr usint MULT_DEPTH = NUMPRIME - 1;
-    { METADATA_UTBGVRNS, "1", {BGVRNS_SCHEME, RING_DIM, MULT_DEPTH, SIZEMODULI, RELIN, BATCH,   DFLT, DFLT,  MAX_DEPTH, FIRST_MOD_SIZE, SEC_LVL, BV,     FLEXIBLEAUTO, DFLT,    PTM,   DFLT,     DFLT,      DFLT,       DFLT, DFLT}, },
-    { METADATA_UTBGVRNS, "2", {BGVRNS_SCHEME, RING_DIM, MULT_DEPTH, SIZEMODULI, RELIN, BATCH,   DFLT, DFLT,  MAX_DEPTH, FIRST_MOD_SIZE, SEC_LVL, BV,     FIXEDMANUAL,  DFLT,    PTM,   DFLT,     DFLT,      DFLT,       DFLT, DFLT}, },
-    { METADATA_UTBGVRNS, "3", {BGVRNS_SCHEME, RING_DIM, MULT_DEPTH, SIZEMODULI, RELIN, BATCH,   DFLT, DFLT,  MAX_DEPTH, FIRST_MOD_SIZE, SEC_LVL, BV,     FIXEDAUTO,    DFLT,    PTM,   DFLT,     DFLT,      DFLT,       DFLT, DFLT}, },
-    { METADATA_UTBGVRNS, "4", {BGVRNS_SCHEME, RING_DIM, MULT_DEPTH, SIZEMODULI, RELIN, BATCH,   DFLT, DFLT,  MAX_DEPTH, FIRST_MOD_SIZE, SEC_LVL, HYBRID, FLEXIBLEAUTO, DFLT,    PTM,   DFLT,     DFLT,      DFLT,       DFLT, DFLT}, },
-    { METADATA_UTBGVRNS, "5", {BGVRNS_SCHEME, RING_DIM, MULT_DEPTH, SIZEMODULI, RELIN, BATCH,   DFLT, DFLT,  MAX_DEPTH, FIRST_MOD_SIZE, SEC_LVL, HYBRID, FIXEDMANUAL,  DFLT,    PTM,   DFLT,     DFLT,      DFLT,       DFLT, DFLT}, },
-    { METADATA_UTBGVRNS, "6", {BGVRNS_SCHEME, RING_DIM, MULT_DEPTH, SIZEMODULI, RELIN, BATCH,   DFLT, DFLT,  MAX_DEPTH, FIRST_MOD_SIZE, SEC_LVL, HYBRID, FIXEDAUTO,    DFLT,    PTM,   DFLT,     DFLT,      DFLT,       DFLT, DFLT}, },
+    // TestType,          Descr, Scheme,      RDim,     MultDepth,  SFBits,     RWin,     BatchSz, Mode, Depth, MDepth,    ModSize,        SecLvl,  KSTech, RSTech,          LDigits, PtMod, StdDev,   EvalAddCt, EvalMultCt, KSCt, MultTech
+    { METADATA_UTBGVRNS, "1", {BGVRNS_SCHEME, RING_DIM, MULT_DEPTH, SIZEMODULI, BV_RELIN, BATCH,   DFLT, DFLT,  MAX_DEPTH, FIRST_MOD_SIZE, SEC_LVL, BV,     FLEXIBLEAUTO,    DFLT,    PTM,   DFLT,     DFLT,      DFLT,       DFLT, DFLT}, },
+    { METADATA_UTBGVRNS, "2", {BGVRNS_SCHEME, RING_DIM, MULT_DEPTH, SIZEMODULI, BV_RELIN, BATCH,   DFLT, DFLT,  MAX_DEPTH, FIRST_MOD_SIZE, SEC_LVL, BV,     FLEXIBLEAUTOEXT, DFLT,    PTM,   DFLT,     DFLT,      DFLT,       DFLT, DFLT}, },
+    { METADATA_UTBGVRNS, "3", {BGVRNS_SCHEME, RING_DIM, MULT_DEPTH, SIZEMODULI, BV_RELIN, BATCH,   DFLT, DFLT,  MAX_DEPTH, FIRST_MOD_SIZE, SEC_LVL, BV,     FIXEDMANUAL,     DFLT,    PTM,   DFLT,     DFLT,      DFLT,       DFLT, DFLT}, },
+    { METADATA_UTBGVRNS, "4", {BGVRNS_SCHEME, RING_DIM, MULT_DEPTH, SIZEMODULI, BV_RELIN, BATCH,   DFLT, DFLT,  MAX_DEPTH, FIRST_MOD_SIZE, SEC_LVL, BV,     FIXEDAUTO,       DFLT,    PTM,   DFLT,     DFLT,      DFLT,       DFLT, DFLT}, },
+    { METADATA_UTBGVRNS, "5", {BGVRNS_SCHEME, RING_DIM, MULT_DEPTH, SIZEMODULI, RELIN,    BATCH,   DFLT, DFLT,  MAX_DEPTH, FIRST_MOD_SIZE, SEC_LVL, BV,     FLEXIBLEAUTO,    DFLT,    PTM,   DFLT,     DFLT,      DFLT,       DFLT, DFLT}, },
```

Comment:
Do we want BV_RELIN to be used here? or should we use HYBRID instead of BV for this line?

Reply:
Yeah, this should be HYBRID. I'm updating it.

Reply:
Updated

Reply:
Updated

Reply:
Updated